plugins/woocommerce/tests/e2e-pw/tests/merchant/products/block-editor/disable-block-product-editor.spec.js

```diff
@@ -18,15 +18,15 @@ let isNewProductEditorEnabled = false;
 const isTrackingSupposedToBeEnabled = !! process.env.ENABLE_TRACKING;
 
 async function dismissFeedbackModalIfShown( page ) {
-	if ( ! isTrackingSupposedToBeEnabled ) {
-		// no modal should be shown, so don't even look for button
-		return;
-	}
+	// if ( ! isTrackingSupposedToBeEnabled ) {
+	// 	// no modal should be shown, so don't even look for button
+	// 	console.log('Feedback modal not shown');
+	// 	return;
+	// }
```

Comment:
This check isn't really necessary, because the following `try` will let this fail gracefully if the modal isn't displayed.  This seems to be the reason why the test is failing on our daily runs.

Reply:
Should we remove this or add a note around why it's commented out?